volunteer-communication-center/tests/blank.js

```diff
@@ -0,0 +1,3 @@
+exports.handler = function(context, event, callback) {
+  callback(null, {});
```

Comment:
Are you going to be writing tests for this app? There is a lot of functionality to it and it would be great to see all the different failure scenarios tested and proven to be working as expected